SConstruct

```diff
@@ -54,6 +83,8 @@ AddOption("--raw", default=os.path.join(root, "raw"), help="Path to raw data")
 AddOption("--repo", default=os.path.join(root, "DATA"), help="Path for data repository")
 AddOption("--calib", default=os.path.join(root, "CALIB"), help="Path for calib repository")
 AddOption("--rerun", default="ci_hsc", help="Rerun name")
+AddOption("--enable-profile", nargs="?", const="profile", dest="enable_profile",
+          help="Profile base name (Note: this is for profiling the scripts, while --profile is for scons)")
```

Comment:
"base name" == "base file name"? Maybe make that clearer here, e.g., "the profile output will be written to base-NUMBER-SCRIPT.pstats in the root directory" or something.